src/classes/Router.mjs

```diff
@@ -1,11 +1,18 @@
 import Router from 'koa-router'
 import config from '../config'
+import StatusCode from './StatusCode'
 
 const router = new Router()
+const documentationUrl = 'https://github.com/FuelRats/FuelRats-API-Docs/blob/master/beta.md'
```

Comment:
This could probably be turned into an ENV variable since there will probably come a time where beta/dev documentation will differ from prod documentation